assets/setup-wizard/features/use-actions-navigator.js

```diff
@@ -38,20 +63,31 @@ const useActionsNavigator = ( actions ) => {
 			return;
 		}
 
-		// Run action.
-		Promise.all( [
-			minimumTimerPromise(),
-			actions[ currentAction ]?.action?.(),
-		] ).then( () => {
-			if ( currentAction + 1 < actions.length ) {
-				setCurrentAction( ( prev ) => prev + 1 );
-			}
-		} );
-	}, [ currentAction, actions ] );
+		const clearAction = runAction();
+
+		return () => {
+			clearAction?.();
+		};
+	}, [ currentAction, runAction ] );
+
+	const errorActions = error && [
```

Comment:
I was playing with `variant: 'primary'` here to see if it adds a bit more contrast between the button text and the error. It might do, but the primary buttons probably pop more than we want. 

<img width="276" alt="Screen Shot 2022-10-07 at 2 54 20 PM" src="https://user-images.githubusercontent.com/68693/194570499-bb997b5e-0be7-4a37-9190-3f995b2b9386.png">

<img width="275" alt="Screen Shot 2022-10-07 at 2 54 34 PM" src="https://user-images.githubusercontent.com/68693/194570440-c4b3ccef-ae7d-425f-93a4-1ec29bf7a4d0.png">

Reply:
<img width="666" alt="Screen Shot 2022-10-07 at 11 21 30" src="https://user-images.githubusercontent.com/876340/194575995-de7f7900-e59b-49f2-9b70-fd0fe0202a78.png">

WDYT about making only the Retry button as `primary`, since it's probably the main action here?

Reply:
That works for me!